### PR TITLE
Custom Icons for Manual Leveling

### PR DESCRIPTION
**Pull request by tome9111991**
I try to make my first Pull request ;P

I have some custom icons for Manual Leveling by modfy the arrows

looks:

Files .jpg & .ICO:
[icons.zip](https://github.com/Jyers/Marlin/files/6377711/icons.zip)

Small Bug:
Center Icons looks like not centered i dont know why.

### CODE REVIEW

File: Marlin/src/lcd/extui/lib/creality_dwin/creality_dwin.cpp [hunk 1]
```diff
@@ -3396,7 +3396,7 @@ void CrealityDWINClass::Menu_Item_Handler(uint8_t menu, uint8_t item, bool draw/
             else {
               if (!mesh_conf.mesh_step_warning && !mesh_conf.goto_mesh_value) {
                 CrealityDWIN.Confirm_Handler((char*)"Z-value not in position");
-                mesh_conf.mesh_step_warning = true;
+                mesh_conf.  mesh_step_warning = true;
```
Comment (tome9111991) [inline Marlin/src/lcd/extui/lib/creality_dwin/creality_dwin.cpp]: miss click

**Comment by Jyers** [comment]
You'll also need to add the icons to the display firmware folder, compile the .ico file in the new section etc.

Looks great btw!

**Comment by tome9111991** [comment]
Icons should be added with Pull request

but you have to uncomment   #define CREALITY_DWIN_EXTUI_CUSTOM_ICONS

if you dont uncomment it will use default icon

**Comment by mhotar** [comment]
Hello. You are doing nice things here, I like it alot.
Thanks to your commits here, I have seen that it is possible to change DWIN icons . I have made some animated main page icons. I will attach them here for you to test, just in case you are interested.
myico
[9.zip](https://github.com/Jyers/Marlin/files/6417571/9.zip)

Thank you for this nice piece of firmware.

**Comment by Jyers** [comment]
Very interesting, I believe the DWIN display has some animation support. I'll look into adding these as a icon pack option

**Pull request merged**